data/product.d/centos-stream.conf

```diff
@@ -0,0 +1,29 @@
+# Anaconda configuration file for CentOS Stream.
+
+[Product]
+product_name = CentOS Stream
```

Comment:
Please, extend the `rhel_products_test` method in `product_test.py`, so the file can be validated by unit tests.